macros/manual_change.ngc

```diff
@@ -46,10 +46,11 @@ O400 endif
 F #<_hal[probe.ps_searchvel]>
 G91
 G38.2 Z #<_ini[TOOLSENSOR]MAXPROBE>
-G0 Z0.26
+G38.4 Z2
+G0 Z0.5
```

Comment:
This isn't strictly a 2.8 change - I found just lifting 0.26 up wasn't always enough, so instead I updated this to probe in the opposite direction, then go a little bit more again.